RegenAllByteCodeNoBuild.cmd

```diff
@@ -0,0 +1,62 @@
+::-------------------------------------------------------------------------------------------------------

```

Comment:
Yeah I agree with this. I think both the refactoring and the script move are good. 
Added https://github.com/Microsoft/ChakraCore/issues/3330